File: 7-Geometria/c/2840_balao++.c
#include <stdio.h>
#include <math.h>

int main () {
    int n, r;
    float pi = 3.1415, v;

    scanf("%d %d", &n, &r);

    v = 1.333333 * pi*pow(n, 3);
    printf("%d\n", (int)floor(r/v));

    return 0;
}

File: 7-Geometria/javascript/2840_balao++.js
var entrada = (require("fs").readFileSync("/dev/stdin", "utf8")).split(' ');
var n = parseInt(entrada[0]);
var r = parseInt(entrada[1]);
var pi = 3.1415;
var v = (4/3) * (pi * (n ** 3));
console.log(`${Math.floor(r/v)}`);
